site/multiDim/MultiDimDataPageContent.tsx

```diff
@@ -278,17 +278,27 @@ export const MultiDimDataPageContent = ({
         }
 
         if (!grapherConfigIsReady) return baseConfig
+        const variables = currentView?.indicators?.["y"]
+        const editUrl =
+            variables?.length === 1
+                ? `variables/${variables[0]}/config`
+                : undefined
```

Comment:
Hm, this will mean that if there is more than a single y variable, the URL will _still_ be wrong...

Reply:
Yeah - do you think it's worthwhile turning editUrl into an object so we can communicate "do not show any edit link"?